### PR TITLE
fix(plugin-meeting): fix 1:1 SIP calls share resolution by disabling …

### PR DESCRIPTION
**Pull request by arun3528**
Disable transcoding and test linus call with screen share
Fixes #[INSERT LINK TO ISSUE NUMBER]
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-175332
---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting/request.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       ensureConversation: ensureConversation || false,
       supportsNativeLobby: 1,
       clientMediaPreferences: {
-        preferTranscoding: preferTranscoding ?? true
+        preferTranscoding
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:66]: Could you add the jsdoc for this option?

File: packages/node_modules/@webex/plugin-meetings/src/meeting/request.js [hunk 2]
```diff
@@ -321,7 +321,7 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       respOnlySdp: true,
       localMedias: options.localMedias,
       clientMediaPreferences: {
-        preferTranscoding: options.preferTranscoding ?? true
+        preferTranscoding: options.preferTranscoding
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:324]: Could you add the jsdoc for this option?

File: packages/node_modules/@webex/plugin-meetings/src/meeting/request.js [hunk 3]
```diff
@@ -36,6 +37,7 @@ export default class MeetingRequest extends StatelessWebexPlugin {
    * @param {boolean} options.pin
    * @param {boolean} options.moveToResource
    * @param {Object} options.roapMessage
+   * @param {Object} options.preferTranscoding send boolean value for transcoding
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:40]:
```suggestion
   * @param {boolean} options.preferTranscoding requests transcoding service from the Webex servers
```

File: packages/node_modules/@webex/plugin-meetings/src/meeting/request.js [hunk 4]
```diff
@@ -61,12 +63,15 @@ export default class MeetingRequest extends StatelessWebexPlugin {
...
-      supportsNativeLobby: 1,
-      clientMediaPreferences: {
-        preferTranscoding: preferTranscoding ?? true
-      }
+      supportsNativeLobby: 1
     };
 
+    if (isBoolean(preferTranscoding)) {
+      body.clientMediaPreferences = {
+        preferTranscoding
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:71]: What does the server prefer? Null or false? We could easily default the value to `false` in the destructuring on line 45.

```js
const {preferTranscoding = false} = options;
```

Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:71]: They were ok with both

**Pull request closed without merging**